pac4j-saml/src/main/java/org/pac4j/saml/metadata/MetadataResolverFactory.java

```diff
@@ -0,0 +1,10 @@
+package org.pac4j.saml.metadata;
+
+import org.opensaml.saml.metadata.resolver.impl.AbstractBatchMetadataResolver;
+import org.springframework.core.io.Resource;
+
+public interface MetadataResolverFactory {

```

Comment:
Please rename the component to be `SAML2MetadataResolverFactory` to keep a consistent naming convention.